### ISSUE TITLE
Oculus Medium 2.0 Fatal Error - Crashing upon start

### ISSUE THREAD

**Issue by smetaucef** [LibreVR/Revive#1054]
I recently purchased the HTC vive and bought oculus medium. It has been crashing since the beginning with a fatal error message. I switched to Steam VR beta and still no use.

Am I missing anything?
 
Thanks

**Comment by Armada651** [comment]
Do you have the latest version?

**Comment by smetaucef** [comment]
Yes, medium crashes immediately with an error or it just stays as a white
screen when I start it.

 I tried using steam VR beta (as well as the other iterations in the beta
list) with different versions of revive.

Is there a specific combo between the revive and steam vr programs that i
need to follow?

I even uninstalled my antivirus and kept only windows defender.

I have a 7700k intel + GTX 1070 so i dont think its a hardware issue.
